vector/src/main/java/im/vector/app/features/home/HomeActivityViewModel.kt

```diff
@@ -183,33 +227,50 @@ class HomeActivityViewModel @AssistedInject constructor(
     private fun observeInitialSync() {
         val session = activeSessionHolder.getSafeActiveSession() ?: return
 
-        session.syncStatusService().getSyncStatusLive()
+        session.syncService().getSyncRequestStateLive()
                 .asFlow()
                 .onEach { status ->
                     when (status) {
-                        is SyncStatusService.Status.InitialSyncProgressing -> {
+// <<<<<<< HEAD
```

Comment:
they have deleted the checkBootstrap here

Reply:
keep element code and update `hasCheckBootstrap` value with `BuildConfig.ENABLE_CROSS_SIGNING`